static/js/components/forms/RegisterDetailsForm.js

```diff
@@ -1,41 +1,287 @@
-        </label>
-        <ErrorMessage name="password" component="div" />
-        <button type="submit" disabled={isSubmitting || !isValid}>
-          Register
-        </button>
+        <div className="form-group">
+          <label htmlFor="legal_address.last_name" className="font-weight-bold">
+            Legal Name*
+          </label>
+          <Field type="text" name="name" className="form-control" />
+          <ErrorMessage name="name" component={FormError} />
+        </div>
+        <div className="form-group">
+          <label htmlFor="legal_address.first_name" className="row">
+            <div className="col-4 font-weight-bold">First Name*</div>
+            <div className="col-8">(Name that will appear on emails)</div>
+          </label>
+          <Field
+            type="text"
+            name="legal_address.first_name"
+            className="form-control"
+          />
+          <ErrorMessage name="legal_address.first_name" component={FormError} />
+        </div>
+        <div className="form-group">
+          <label htmlFor="legal_address.last_name" className="font-weight-bold">
+            Last Name*
+          </label>
+          <Field
+            type="text"
+            name="legal_address.last_name"
+            className="form-control"
+          />
+          <ErrorMessage name="legal_address.last_name" component={FormError} />
+        </div>
+        <div className="form-group">
+          <label htmlFor="password" className="font-weight-bold">
+            Password*
+          </label>
+          <Field type="password" name="password" className="form-control" />
+          <ErrorMessage name="password" component={FormError} />
+          <div className="label-secondary">
+            Password must contain:
+            <ul>
+              <li>at least 8 characters</li>
+              <li>at least 1 number and 1 letter</li>
+            </ul>
+          </div>
+        </div>
+        <div className="form-group">
+          {/* LegalAddress fields */}
+          <label htmlFor="legal_address.last_name" className="font-weight-bold">
+            Street Address*
+          </label>
+          <FieldArray
+            name="legal_address.street_address"
+            render={arrayHelpers => (
+              <div>
+                {values.legal_address.street_address.map((line, index) => (
+                  <div key={index}>
+                    <Field
+                      name={`legal_address.street_address[${index}]`}
+                      className="form-control row-inner"
+                    />
+                    {index === 0 ? (
+                      <ErrorMessage
+                        name="legal_address.street_address"
+                        component={FormError}
+                      />
+                    ) : null}
+                  </div>
+                ))}
+                {values.legal_address.street_address.length < 5 ? (
+                  <div
```

Comment:
For accessibility reasons, this should be a `<button>` because a screen reader will not know the user can click on a `<div>` without adding a bunch of extra attributes and event handlers.

Reply:
Abdul preferred it to look more like a link than a button. The style includes `cursor:pointer`.  Okay with you if I change it to a button but style it to look like a link?